project/Deps.scala

```diff
@@ -177,6 +182,7 @@ object Deps {
     val junitInterface =
       "com.novocode" % "junit-interface" % V.junitV % "test" withSources () withJavadoc ()
     val logback = Compile.logback % "test"
+    val grizzledSlf4j = Compile.grizzledSlf4j % "test"
```

Comment:
Hmm.. is this a `test` dependecy? If so, it should live in `Test`. But my understanding is that we want this available in our source code to... which means we should probably investigate what deps that `grizzleSlf4j` is pulling in. Hopefully nothing...

Reply:
Looks like it is ! 

>[info] org.bitcoin-s:bitcoin-s-core_2.13:0.3.0+452-0160ce12-SNAPSHOT [S]
[info]   +-org.bitcoin-s:bitcoin-s-crypto_2.13:0.3.0+452-0160ce12-SNAPSHOT [S]
[info]   | +-org.bitcoin-s:bitcoin-s-secp256k1jni:0.3.0+452-0160ce12-SNAPSHOT
[info]   | | +-org.scijava:native-lib-loader:2.3.4
[info]   | |   +-org.slf4j:slf4j-api:1.7.25 (evicted by: 1.7.30)
[info]   | |   +-org.slf4j:slf4j-api:1.7.30
[info]   | |
[info]   | +-org.bouncycastle:bcprov-jdk15on:1.66
[info]   | +-org.scodec:scodec-bits_2.13:1.1.20 [S]
[info]   |
[info]   +-org.bouncycastle:bcprov-jdk15on:1.66
[info]   +-org.clapper:grizzled-slf4j_2.13:1.3.4 [S]
[info]   | +-org.slf4j:slf4j-api:1.7.30
[info]   | +-org.slf4j:slf4j-api:1.7.9 (evicted by: 1.7.30)
[info]   |
[info]   +-org.scodec:scodec-bits_2.13:1.1.20 [S]
[info]
[info] ()
[success] Total time: 0 s, completed Sep 15, 2020 2:29:12 PM